### ISSUE TITLE
Force X11 doesn't work

### ISSUE THREAD

**Issue by cuihaoleo** [rcaelers/workrave#525]
**Describe the bug**
"Force the use of X11 on Wayland" setting doesn't seem to effect.

**To Reproduce**
1. Build workrave (I had to disable GTK4 due to #524 )
2. Run workrave on KDE wayland
3. Check if it runs on Wayland or X11

"Force the use of X11 on Wayland" setting is by default on. But it has no effect. See the screenshot below. You can tell from the title bar icon that it still uses Wayland.

**Expected behavior**

"Force the use of X11 on Wayland" setting works.

**Screenshots**
![image](https://github.com/rcaelers/workrave/assets/889871/4689c78b-985e-410f-9551-a58dab2b0bbd)

** Linux (please complete the following information in case you encountered the bug on Linux):**
 - Distribution: Arch Linux
 - Distribution Version: rolling
 - Desktop Environment: KDE Plasma 5.27.10, Wayland session
 - Workrave Version: Beta 10 / Git HEAD (330441054853c0c71c7830cc405c6681f95557f2)

**Additional context**

Log when runs from terminal:
```
[2024-02-07 11:14:23.895] [workrave] [info ] Workrave started
[2024-02-07 11:14:23.895] [workrave] [info ] Log file: /home/cuihao/.cache/workrave/workrave.log
[2024-02-07 11:14:23.979] [toolkit] [info ] Toolkit reported # displays : 2
[2024-02-07 11:14:23.979] [toolkit] [info ] Display #0: primary=false x=0 y=0 w=1920 h=1080

(workrave:817772): Gtk-CRITICAL **: 11:14:23.980: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
[2024-02-07 11:14:23.981] [workrave] [info ] created plugin workrave.GenericDBusApplet
[2024-02-07 11:14:23.981] [workrave] [info ] created plugin workrave.GnomeSession
[2024-02-07 11:14:23.981] [workrave] [info ] created plugin workrave.X11SystrayAppletWindow
[2024-02-07 11:14:23.981] [workrave] [info ] created plugin workrave.IndicatorAppletMenu
```

**Comment by cuihaoleo** [comment]
`GDK_BACKEND=x11 /usr/bin/workrave` will run it on X11 backend.

**Comment by rcaelers** [comment]
Fixed in 1.11 beta11
